tests/test_hintfonts.py

```diff
@@ -21,9 +22,13 @@ def __init__(self, inpath, outpath):
         self.verbose = False
 
 
+def _temp(tmpdir, path):
```

Comment:
the name `_temp` can be misleading. that code is not making a temporary directory, is just concatenating a py.path.local object with another and returning a str. The tmpdir fixture is responsible for creating the temporary directory and cleaning it up automatically after the test function has completed. I think factoring out this one liner only obfuscates things -- well, at least for me, who am familiar with using the fixture and py.path.local.

Reply:
maybe `_join` or `_pjoin` is a better name